api/templates/design.go

```diff
@@ -567,6 +568,11 @@ var TemplateRequest = Type("TemplateRequest", func() {
 	Required("template")
 })
 
+var TemplatesRequest = Type("TemplatesRequest", func() {
+	Extend(RepositoryRequest)
+	Attribute("filter", String)
```

Comment:
Shouldn't it be an enum rather than a string? From the code it seems that values other than `"keboola.data-apps"` and `"keboola.components"` don't do anything.

Reply:
Good question, It would be nice that these values are in the API docs and generated by `goa` so we don't need to create separate documentation.
We currently do not know about other values

Reply:
[aa1c10b](https://github.com/keboola/keboola-as-code/pull/2144/commits/aa1c10bc266644f5a140cdf1dee92f7384a3c629) - enum+description

[f1bab1a](https://github.com/keboola/keboola-as-code/pull/2144/commits/f1bab1a7f6069d9ce708967def31d0a63332cdbb) - update test

Reply:
@jachym-tousek-keboola it was a good catch 👍

Reply:
Much better, thank you!